src/dc_receive_imf.rs

```diff
@@ -1021,39 +1021,32 @@ fn create_or_lookup_group(
     }
 
     // add members to group/check members
-    // for recreation: we should add a timestamp
     if recreate_member_list {
-        // TODO: the member list should only be recreated if the corresponding message is newer
-        // than the one that is responsible for the current member list, see
-        // https://github.com/deltachat/deltachat-core/issues/127
-
-        let skip = X_MrRemoveFromGrp.as_ref();
-        sql::execute(
-            context,
-            &context.sql,
-            "DELETE FROM chats_contacts WHERE chat_id=?;",
-            params![chat_id],
-        )
-        .ok();
-        if skip.is_none() || !addr_cmp(&self_addr, skip.unwrap()) {
+        if !chat::is_contact_in_chat(context, chat_id, DC_CONTACT_ID_SELF) {
             chat::add_to_chat_contacts_table(context, chat_id, DC_CONTACT_ID_SELF);
         }
         if from_id > DC_CHAT_ID_LAST_SPECIAL
```

Comment:
Unrelated, but this one should be `DC_CONTACT_ID_LAST_SPECIAL`

Reply:
true.